build.gradle

```diff
@@ -54,7 +54,7 @@ def coreProjects = subprojects.findAll { !['util'].contains(it.name) }
 def javaProjects = subprojects.findAll { !it.name.contains('scala') }
 def scalaProjects = subprojects.findAll { it.name.contains('scala') }
 def javaMainProjects = javaProjects.findAll { !['util'].contains(it.name) }
-def javaCodeCheckedProjects = javaMainProjects.findAll { !['driver-benchmarks'].contains(it.name) }
+def javaCodeCheckedProjects = javaMainProjects.findAll { !['driver-benchmarks', 'driver-workload-executor'].contains(it.name) }
```

Comment:
Perhaps there's a cleaner way to do this?

Reply:
I think this is fine for now, as it allows for centralised  configuration of projects in the main build.

A cleaner alternative would potentially to separate out the logic for code checking into a separate file and only import it in the projects that actually need them.  Then configuration would essentially be contained on a per project basis.  I think when we move to kotlin for the build that would be the ideal time to do that work.